docs/docs/unreachable.md

```diff
@@ -0,0 +1,143 @@
+return of the lost task, it will be killed immediately after it is seen by the system (assuming it is past expunge time).
+
+When the unreachable strategy is `{0, 0}`, the replacement is nearly immediate, with a replacement commonly occurring (for the test app) within roughly 3 seconds, and the expunge happening within 8-11 seconds. When we include this with the Mesos details described above for the loss of 1 agent, it means that Marathon will replace a task as soon as it is notified that the task is unreachable (which happens when the Mesos Master marks an agent as inactive), and it will expunge the original task as soon as it is reachable. If there is time configured for the strategy, then the event cycle that Marathon uses to managing if a task should be replaced or expunged is the task reconciliation cycle. The importance of this is 2 fold: 1) the unreachable strategy time has to expire and 2) the next task reconciliation needs to happen. The task reconciliation is a Marathon system configuration [`--reconciliation_interval`](https://mesosphere.github.io/marathon/docs/command-line-flags.html). The Marathon defined default is 10 minutes. This means that an unreachable strategy which includes `inactiveAfterSeconds` = 60, will have a task replaced between 60 seconds and 11 minutes. The reconciliation interval has the same effect on the expunging of a task.
+
+## DC/OS Unreachable Task Scenarios
+
+The follow scenarios a given for a task going unreachable given a default configuration of a DC/OS cluster. This also assumes that there are resources with required constraints available in the cluster for the replacement task. All marked time references are from the actual loss of an agent from the cluster.
+
+### Scenario 1: UnreachableStrategy `{0, 0}`, 1 agent becomes unresponsive for 4 minutes
+
+- 12:00:00 - Agent 1 stops responding due to a network partition (and is running one task).
+- 12:04:00 - The network partition is resolved, and agent 1 starts responding again.
+
+At a task level, Marathon is not made aware that the agent was momentarily not responding; Mesos does not report the  task as unreachable.
+
+### Scenario 2: UnreachableStrategy `{0, 0}`, 1 agent becomes unresponsive for 6 minutes
+
+- 12:00:00 - Agent 1 stops responding due to a network partition (and is running one task).
+- 12:05:00 - The Mesos master marks Agent 1 as inactive, and publishes a `TASK_UNREACHABLE` status update.
+- 12:05:03 - Shortly after, Marathon replaces 1 tasks for with the `TASK_UNREACHABLE` status was received. Record of the unreachable task is quickly expunged.
+- 12:06:00 - Agent 1 starts responding again. The Mesos master marks the agent as reachable, and sends a `TASK_RUNNING`status update.
+- 12:06:03 - Marathon kills the previously unreachable task kill.
```

Comment:
`Marathon kills the previously unreachable task kill` marathon kills the kill? What about: `- Since the original task was already expunged marathon kills it by sending a kill request to mesos`